common/web/lm-worker/src/main/correction/transform-tokenization.ts

```diff
@@ -0,0 +1,51 @@
+import { applyTransform, type Tokenization } from "@keymanapp/models-templates";
+
+export function tokenizeTransform(
```

Comment:
Can you include a function comment?

Reply:
I've gone ahead and done this (and the other similar suggestion) with https://github.com/keymanapp/keyman/pull/11997/commits/1d6ea1aff05acc1bbdac1a78827b0469c20dbe2e on #11997.

The chain is otherwise ready to merge, and that seemed the best way to avoid potential merge conflicts.

Reply:
Yep, sounds good to me